database/db.js

```diff
@@ -1,6 +1,16 @@
 const config = require('config')
 const pgPromise = require('pg-promise')
 const pgp = pgPromise()
-const db = pgp(config.postgres)
+// const db = pgp(config.postgres)
+const db = pgp(`postgres://${process.env.USER}@localhost:5432/grandiose`)
 
-const createProjects =  'INSERT INTO projects (id, project_name) VALUES ($1, $2) RETURNING *'
+
+const createProjects =  'INSERT INTO projects (project_name) VALUES ($1) RETURNING *'
+
+const Projects = {
+  create: (project_name) => {
+    return db.oneOrNone( createProjects, [project_name] )
+  }
+}
```

Comment:
this is a great way to set up the model functionality for Projects.create and so forth